src/Core/OrganizationFeatures/OrgUser/Invitation/Accept/OrganizationUserAcceptAccessPolicies.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
```

Comment:
I _do_ like splitting out access policies like this - I think it's a really good effort to abstract and re-use them, and increase testability. But I have the same concerns about them being excessively compartmentalised into lots of tiny classes. 

To me it would make sense if each command also housed its own access policy logic - although that may be mixing concerns. Or, if we were sticking to services, _OrganizationUserService_ and then _OrganizationUserAccessPolicies_ (for example). Not really sure how I would structure it to be honest, just thinking aloud.